shared/homeless/symfony.lock

```diff
@@ -209,20 +197,6 @@
     "sonata-project/intl-bundle": {
         "version": "2.7.0"
     },
-    "sonata-project/media-bundle": {
```

Comment:
Is it ok to remove this bundle? Can you explain why it was needed before and is not needed now?